ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -108,4 +107,4 @@ cat > packet-setup.yaml <<-EOF
       dest: "${SHARED_DIR}/packet-conf.sh"
 EOF
 
-ansible-playbook packet-setup.yaml -e "packet_hostname=ipi-${NAMESPACE}-${JOB_NAME_HASH}-${BUILD_ID}"  |& gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0; fflush(); }'
+ansible-playbook packet-setup.yaml -e "packet_hostname=${PACKET_ACCOUNT}-${NAMESPACE}-${JOB_NAME_HASH}-${BUILD_ID}"  |& gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0; fflush(); }'
```

Comment:
What's the real requirement to change the name? If not, `$PACKET_ACCOUNT` could be added to to the [`tags`](https://github.com/openshift/release/blob/7964b0e68f092aa331bd72db2f0d2745f286aef1/ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh#L42)

Reply:
Not all the packet instances are ipi related

Reply:
Makes sense, especially considering that there could be further split (and originally we added the `ipi` prefix to discriminate across the other machines, so a similar situation). It could be then even removed if it's clear from the context, and the instance name will be shorten then (which is a good thing especially when troubleshooting in the Packet UI). In any case I think it would be useful to add it in the tags, as it could help in filtering and/or checking.

Reply:
I changed the name with the idea of easily discerning the packet servers in the Slack status channel. Since the hostname is in the error message